.github/workflows/pullrequest.yml

```diff
@@ -469,7 +489,9 @@ jobs:
         run: ./scripts/ci/run-in-parallel-native.sh build
 
   success:
-    runs-on: kubernetes-runner
+    runs-on: ec2-runners
+    container:
+      image: public.ecr.aws/m3u4c4h9/island-is/actions-runner-public:latest
     if: always()
     needs:
       - prepare
```

Comment:
shellcheck job is not on the list and is therefore ignored. Also needs to be added to the steps similarly to the other ones.